js/kucoinfutures.js

```diff
@@ -1253,9 +1256,14 @@ module.exports = class kucoinfutures extends kucoin {
         await this.loadMarkets ();
         // only fetches one balance at a time
         const request = {};
-        const defaultCode = this.safeString (this.options, 'defaultCode');
-        const code = this.safeString2 (params, 'code', 'currency', defaultCode);
-        if (code !== undefined) {
+        const currencyId = this.safeString (params, 'currency');
+        let defaultCode = this.safeString (this.options, 'code');
```

Comment:
I think that it should be `defaultCode`, it's more obvious to the user what that would be